cli/tsc.rs

```diff
@@ -1725,7 +1712,7 @@ mod tests {
     let source_code = compiled_file.code;
     assert!(source_code
       .as_bytes()
-      .starts_with(b"console.log(\"Hello World\");"));
+      .starts_with(b"console.log('Hello World');"));
```

Comment:
The output defaults to different quotes?

Reply:
Super yucky!  It really shouldn't be doing that.

Reply:
In general the output from codegen is rather poorly formatted. I guess we can run it through prettier before caching to disk...

Reply:
Well, swc_ecma_codegen sucks for a linter because it's implemented based on usage for the compiler.
I'll fix it asap.

Reply:
Fixed after upgrade to latest `swc_ecma_codegen`